flybywiresim/a32nx
# build(scripts): only prune if image isn't available

## Summary of Changes
Only prune the docker system when the image isn't available which only happens if it's a first-time use or we are using a new image. The pull is done automatically by  `docker run` hence I removed the command.

## Screenshots (if necessary)
N/A

## References
N/A

## Additional context
N/A

Discord username (if different from GitHub): nistei#1362

## Testing instructions
Dev Test:
1. `docker system prune --filter label=flybywiresim=true -f`
2. Build the A32NX -> Should download image
3. Build again -> Should use existing image

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
